prettier.config.js

```diff
@@ -7,8 +7,17 @@ module.exports = {
       files: ['docs/**/*.md', 'docs/src/pages/**/*.{js,tsx}', 'docs/data/**/*.{js,tsx}'],
       options: {
         // otherwise code blocks overflow on the docs website
+        // The container is 751px
         printWidth: 85,
       },
     },
+    {
+      files: ['docs/pages/blog/**/*.md'],
+      options: {
+        // otherwise code blocks overflow on the blog website
+        // The container is 676px
+        printWidth: 82,
```

Comment:
Based on https://deploy-preview-33648--material-ui.netlify.app/blog/mui-x-v5/#simplified-style-customization, we might need one more char

```suggestion
        printWidth: 81,
```